src/enzyme_ad/jax/RegistryUtils.cpp

```diff
@@ -195,3 +270,66 @@ void prepareRegistry(mlir::DialectRegistry &registry) {
   mlir::registerLLVMDialectImport(registry);
   mlir::registerNVVMDialectImport(registry);
 }
+
+void initializePasses(mlir::DialectRegistry &registry) {
```

Comment:
this shouldn't take a registry here (and also be called once during initialization)

Reply:
during initialization of what?

Reply:
Of the binary itself. Like enzymexlamliropt ran it during init, same with reactant’s init. These need to be run once per program, not dialect